src/core/useDetectGPU.tsx

```diff
@@ -1,12 +1,8 @@
 import * as React from 'react'
 import { getGPUTier, GetGPUTier, TierResult } from 'detect-gpu'
+import { useAsset } from 'use-asset'
 
 export function useDetectGPU(props?: GetGPUTier): TierResult | null {
```

Comment:
would this make the return value of this hook incorrect? i.e. it won't return null because it's suspended?

Reply:
you're right!